### ISSUE TITLE
Next.js webpack configuration detection failed with the following error TypeError [ERR_INVALID_ARG_TYPE]: The "paths[0]" argument must be of type string. Received undefined

### ISSUE THREAD

**Issue by jantimon** [depcheck/depcheck#880]
### Bug Description

**Next.js Webpack Configuration Detection Fails in Depcheck**

When using `depcheck` with a Next.js project, the detection of the webpack configuration fails. The error encountered is a `TypeError [ERR_INVALID_ARG_TYPE]`, stating that the `"paths[0]" argument must be of type string, but received undefined`. This issue seems to stem from the way Next.js handles the `.context` of a webpack config, which is always a filepath. This mismatch in expectations causes `depcheck` to fail in processing the webpack configuration correctly.

### Code Snippets

Where the dependency is used:
```js
// next.config.js
const { withYak } = require("next-yak/withYak");

const nextConfig = {
  // your next.js config
};

module.exports = withYak(nextConfig);
```

### Versions

- `node -v`: 20.9.0
- `depcheck --version`: 1.4.7

### Possible fix:

I tried the following in our codebase and was able to fix the bug:

```diff
diff --git a/node_modules/depcheck/dist/special/webpack.js b/node_modules/depcheck/dist/special/webpack.js
index 7544ccc..6476a55 100644
--- a/node_modules/depcheck/dist/special/webpack.js
+++ b/node_modules/depcheck/dist/special/webpack.js
@@ -102,6 +102,7 @@ function parseWebpackConfig(webpackConfig, deps) {
 }
 async function loadNextWebpackConfig(filepath) {
   const fakeConfig = {
+    context: process.cwd(),
     plugins: [],
     module: {
       rules: []
```
